cypress/utils/dashboards/datasource-management-dashboards-plugin/commands.js

```diff
@@ -29,6 +29,29 @@ Cypress.Commands.add('deleteAllDataSources', () => {
   );
 });
 
+Cypress.Commands.add('createDataSourceNoAuth', () => {
+  cy.request({
+    method: 'POST',
+    url: `${Cypress.config('baseUrl')}/api/saved_objects/data-source`,
+    headers: {
+      'osd-xsrf': true,
+    },
+    body: {
+      attributes: {
+        title: `9201`,
```

Comment:
should we use more meaningful name as title?

Reply:
Changed to a more meaningful one here: [46d3442](https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/1146/commits/46d3442825d45ef318d72cd3a45aac3f4c0078da)